src/ripple/nodestore/Database.h

```diff
@@ -94,6 +92,11 @@ class Database : public Stoppable
     virtual std::int32_t
     getWriteLoad() const = 0;
 
+    /** Retrieve backend read and write stats. The return pointer may be null.
+    */
+    virtual Backend::Counters const*
```

Comment:
We're using the pointer like an `optional<Counters&>`, which I like, but we should probably add a comment saying so.